### PR TITLE
WIP: Creation of Elections subproject

### PR DESCRIPTION
**Pull request by jberkus**
Creation of the subproject and initial file reorganization.

Moves all elections files under top-level /elections folder
Adds READMEs for the subproject duties
Adds initial subproject members

This does need to be finished as a prerequisite to the 2022 SC election.  Stuff that we need to finish this PR:

- [ ] Approve all new processes/ownership introduced 
- [ ] Finalize initial list of subproject members/approvers
- [ ] Do build on sigs.yaml

IMPORTANT: the elections app will need to be upgraded in coordination with this file location change.  As such, it should not be merged except as a coordinated activity.  
/hold

This will be the first of several large PRs to organize elections.  Additional work that needs to be done, including:

- Extracting all documentation from the individual election directories and consolidating it into some kind of sensible structure.
- Provide template election files
- Build out more detailed processes for undocumented things like picking EOs.

/sig contributor-experience
/area elections

### CODE REVIEW

**Comment by jberkus** [comment]
oh, yeah:

Fixes #6084

**Comment by jberkus** [comment]
Would like feedback on content from our TLs (and other people) before I fix test issues and build sigs.yaml, thanks!

File: elections/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+maintenance, and security.  As such, new approvers must be approved by
+the [Contributor Experience] chairs or by the Steering Committee.
+
+### Communications
+
+The elections subproject can be reached by the following mechanisms:
+
+* tagging area/elections in issues or PRs in the Kubernetes/Community repo
+* #sig-contribex channel in Kubernetes slack
+* The weekly [Contributor Experience] meeting
```
Comment (mrbobbytables) [inline elections/README.md:47]: Technically the zoom one we have on the calendar is biweekly...thinking about it as a follow up item we should better document the slack standup one we host on alternate weeks.

File: elections/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+### Minor Elections
+
+The subproject will assist Kubernetes teams/SIGs/WGs in preparing any internal
+elections they want to run.  This includes watching for election issues and PRs
+and either helping create the appropriate files or auditing them for the teams.
+The subproject will work with the Contributor Comms team to promote the elections
+as appropriate.
+
+Any named Kubernetes team (see [requesting an election]) may run an election 
+in elections.k8s.io.  As they cost nothing to run, and multiple elections can
```
Comment (detiber) [inline elections/README.md:101]: Is this statement true? I would expect there to be *some* cost associated with running an election, even if it is relatively trivial, since Elekto is running somewhere.

Maybe rewording this a bit to something like:
Since Elekto allows running multiple elections concurrently, the main limitation for running additional elections is the time of Contribex volunteers assist with the process.

File: elections/teams/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Each team election goes in its own folder in the kubernetes/community repo,
+using the path `elections/teams/team-name/election-name`, 
+e.g. `elections/teams/clusterapi/leads-2022`.  This folder should then have
+the following files in it, with complete contents.  See the 
+[Elekto administration docs] for more details.
+
+* election.yaml file defining the election
+* election-desc.md file with a text description of the election
+* voters.yaml with the initial list of voters
```
Comment (mrbobbytables) [inline elections/teams/README.md:63]: Potential follow up item, templates for these files.

**Review by mrbobbytables — COMMENTED** [review]
Thanks @jberkus just a few non-blocking comments and potential follow up items after this is merged. 👍

File: elections/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+Approvers and reviewers are also responsible for the Elections roadmap, 
+maintenance, and security.  As such, new approvers must be approved by
+the [Contributor Experience] chairs or by the Steering Committee.
+
+### Communications
+
+The elections subproject can be reached by the following mechanisms:
+
+* tagging area/elections in issues or PRs in the Kubernetes/Community repo
```
[the remark this answers is not in the document]
Comment (jberkus) [reply elections/README.md:45]: I have to do one more edit run regardless, so we can incorporate all these things.

File: elections/README.md (hunk 1 above)
Comment (jberkus) [reply elections/README.md:47]: Heck, we need to make the slack standup a real thing.  That's on my TODO list as chair, but likely not until August.

File: elections/teams/README.md (hunk 3 above)
Comment (jberkus) [reply elections/teams/README.md:63]: Yes, definitely.  That's also related to fixing this issue in Elekto: https://github.com/elekto-io/elekto/issues/47

**Comment by jberkus** [comment]
All feedback addressed, sigs updated.

@cblecker @nikhita please approve, thanks!

**Comment by cblecker** [comment]
/approve
Wearing my ContribEx TL hat

**Review by mrbobbytables — APPROVED** [review]
/lgtm
thanks @jberkus

**Comment by mrbobbytables** [comment]
We have approval from the two TLs, should be good enough to merge :) @jberkus please release the hold when you're ready
